src/main/resources/anniv5Resources/localization/eng/magnetizepack/Cardstrings.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
It might be a good idea to have at least one discard card in here just to build the association with your mechanic